lib/v2/5eplay/index.js

```diff
@@ -0,0 +1,83 @@
+const cheerio = require('cheerio');
+const zlib = require('zlib');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { getAcwScV2ByArg1 } = require('./utils');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://csgo.5eplay.com/';
+    const apiUrl = `${rootUrl}api/article?page=1&type_id=0&time=0&order_by=0`;
+    const articleUrl = `${rootUrl}article`;
+
+    const { data: response } = await got({
+        method: 'get',
+        url: apiUrl,
+    });
+
+    const expiredTime = 25 * 60 * 1000;
```

Comment:
Please use `config.cache.routeExpire` instead. Note that the unit for all cache-related function is second.
```js
const config = require('@/config').value;
```